google-cloud-language/Rakefile

```diff
@@ -119,19 +135,13 @@ task :ci do
   header "BUILDING google-cloud-language"
   header "google-cloud-language rubocop", "*"
   Rake::Task[:rubocop].invoke
-  header "google-cloud-language doctest", "*"
-  Rake::Task[:doctest].invoke
+  header "google-cloud-language yard", "*"
```

Comment:
Swapped out doctest for YARD in the CI task. Really amazed that the docs build without warning!